configure.ac

```diff
@@ -74,6 +74,12 @@ fi
 AM_CONDITIONAL(ENABLE_TESTS_GLES,
     [test "x$enable_tests_gles" = "xyes"])
 
+HAVE_GTEST="no"
+ifdef([GTEST_LIB_CHECK],
```

Comment:
here is all package from:

autopkgtest - automatic as-installed testing for Debian packages
gtester2xunit - Helper for converting gtester xml output to xunit.
libgtest-dev - Google's framework for writing C++ tests - header files
libxorg-gtest-data - X.Org dummy testing environment for Google Test - data
libxorg-gtest-dev - X.Org dummy testing environment for Google Test - headers
libxorg-gtest-doc - X.org dummy testing environment for Google Test - documentation
autopkgtest-xenlvm - autopkgtest Xen/LVM2 based testbed snapshot system
gnome-desktop-testing - runner for GNOME installed tests
libdist-zilla-plugin-podspellingtests-perl - release tests for POD spelling
gnome-desktop-testing-dbgsym - debug symbols for package gnome-desktop-testing

I installed libgtest-dev, i can see all /usr/include/gtest/gtest.h, but i did find gtest.m4 in my system.
And do we have license issue if we include this .m4 in yami?